assets/wizards/readerRevenue/views/donation/index.tsx

```diff
@@ -169,7 +169,12 @@ const Donation = () => {
 				<>
 					<Card noBorder headerActions>
 						<h2>{ __( 'Donations Landing Page', 'newspack' ) }</h2>
-						<Button variant="secondary" isSmall href={ wizardData.donation_page.editUrl }>
+						<Button
+							variant="secondary"
+							isSmall
+							href={ wizardData.donation_page.editUrl }
+							onClick={ undefined }
```

Comment:
This is not necessary. `onClick` will already be undefined by default on `<Button />`

Reply:
It is, for TypeScript. There's no TS validation set up yet for this repo (blocked by https://github.com/Automattic/newspack-plugin/pull/1713), but once it's in place, supplying this prop will be required. 
The code in `components` is not yet TS-ready, because there's a separate build & distribution process there.

Reply:
Yes, until `components` code is converted to TS.